client/my-sites/plugins/plugins-browser/collection-list-view/index.tsx

```diff
@@ -128,8 +132,17 @@ export default function CollectionListView( {
 	siteSlug: string;
 	sites: any;
 } ): ReactElement | null {
+	const selectedSite = useSelector( getSelectedSite );
+	const isJetpack = useSelector( ( state ) => isJetpackSite( state, selectedSite?.ID ) );
+	const isAtomic = useSelector( ( state ) => isSiteAutomatedTransfer( state, selectedSite?.ID ) );
+	const isJetpackSelfHosted = isJetpack && ! isAtomic;
+
 	const collections = useCollections();
 
+	if ( isJetpackSelfHosted ) {
+		return null;
+	}
```

Comment:
I debated a bit on if we should perform the `isJetpackSelfHosted` check here or up a level where `CollectionListView` is called. I feel like it makes the most sense here because there's currently not a case where we'd want to show a collection to a Jetpack site. Including it here keeps code duplication down, and prevents accidentally displaying the collection to Jetpack if a new reference to `CollectionListView` is created.

Reply:
Yeah, I agree with you @DustyReagan. It seems to me the best place to make this decision. I faced a similar issue recently with the logged-out check, [and the answer was the same](https://github.com/Automattic/wp-calypso/pull/68835#issuecomment-1274812271): do it in the component itself.